routing/hostname_pattern.rst

```diff
@@ -260,8 +260,11 @@ You can also set the host option on imported routes:
 
         use Symfony\Component\Routing\RouteCollection;
 
+        $importedCollection = $loader->import("@AcmeHelloBundle/Resources/config/routing.php");
+        $importedCollection->setHost('hello.example.com');
+        
         $collection = new RouteCollection();
-        $collection->addCollection($loader->import("@AcmeHelloBundle/Resources/config/routing.php"), '', array(), array(), array(), 'hello.example.com');
+        $collection->addCollection($importedCollection);
```

Comment:
I think we don't need to wrap the imported collection in another collection, but can do the following instead (we can then remove the `use` statement too):

```php
$collection = $loader->import("@AcmeHelloBundle/Resources/config/routing.php");
$collection->setHost('hello.example.com');

return $collection;
```